src/test/ui/stability-attribute/rustc-encodable-stability.rs

```diff
@@ -0,0 +1,8 @@
+// edition:2018
+#![allow(deprecated)]
+extern crate rustc_serialize;
```

Comment:
This doesn't seem to be finding rustc_serialize, so I'm not really sure how to interact with this deprecated API in order to test it from a ui test.

Reply:
I think moving the test from `ui` to `ui-fulldeps` is the simplest approach that can work (since it's a compiler crate). Otherwise you'd need to mock it or something.